example.env

```diff
@@ -4,3 +4,5 @@ RESTDAPI_DOCS=http://172.16.38.113:6212/docs
 DEV_PORTAL_SITE=http://testtool.dhq.onem:6060
 
 TODO_APP_GITHUB_PY_URL=https://github.com/mvnm/django-onem-todo
+
+SWAGGER_SCHEMA_URL=https://app.swaggerhub.com/apis/romeo1m/schemajson/m5.0
```

Comment:
I've changed the version prefix. But in order to avoid having the version in the url, you can cut the version, and you'll be redirected to the default version (which should be the latest).

Reply:
Do we keep it under your account or transfer it to `onem-com` ?

Reply:
On my account I have rights to edit :). Ideally this won't be the final address.

Reply:
@chrishornmem let's transfer the schema to onem-com and have Romeo as an admin. What do you think, is it possible to transfer it?

Reply:
Have discussed with Romeo, I'm thinking to just drop swaggerhub and use our docs as the way to share the schema.